Resources/Maps/_Sunrise/planet_prison.yml

```
@@ -7124,55 +6990,10 @@
-          - 0
-          - 0
-          - 0
-          - 0
-          - 0
-        - volume: 2500
-          temperature: 293.14926
+          temperature: 293.15
           moles:
-          - 21.824879
-          - 82.10312
+          - 21.6852
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 12 but found 10

(indentation)